bugsnag-android-core/src/main/java/com/bugsnag/android/ClientInternal.kt

```diff
@@ -316,7 +316,7 @@ internal class ClientInternal constructor(
     private fun onConnectivityChange(hasConnection: Boolean, networkState: String?) {
         leaveAutoBreadcrumb(
             "Connectivity changed", BreadcrumbType.STATE,
-            mapOf(
+            mutableMapOf(
```

Comment:
I can't recall what the conclusion of the discussion we had with @lemnik earlier was - but I think that using `HashMap` would be marginally less of a footprint than `mutableMapOf` due to its use of `LinkedHashMap` behind the scenes.

Reply:
Yes, I wanted to start out with just mutableMapOf and then we could work from that baseline to determine how much (if any) HashMap would gain us. That way if the gains are negligible, we can remain idiomatic.